tensorboard/webapp/widgets/line_chart_v2/sub_view/line_chart_axis_view.scss

```diff
@@ -28,10 +28,30 @@ text {
   user-select: none;
 }
 
+$_icon_size: 16px;
+
 .axis {
   display: flex;
   height: 100%;
   width: 100%;
+
+  .extent-edit-button {
+    --tb-icon-size: #{$_icon_size};
+    height: 24px;
+    line-height: 24px;
+    position: absolute;
+    right: 5px;
+    top: 5px;
+    visibility: hidden;
+    width: 24px;
+    padding: 0;
+
+    mat-icon {
+      height: $_icon_size;
+      width: $_icon_size;
+      line-height: $_icon_size;
```

Comment:
Would it make sense to also include `line-height: var(--tb-icon-size)` in _tb_theme.template.scss?